modules/tinymce/src/plugins/lists/test/ts/browser/ContentEditableFalseActionsTest.ts

```diff
@@ -0,0 +1,96 @@
+  const hook = TinyHooks.bddSetupLight<Editor>({
+    plugins: 'lists',
+    toolbar: 'numlist bullist',
+    base_url: '/project/tinymce/js/tinymce'
+  }, [ Plugin ], true);
+
+  it('TINY-8920: List toolbar buttons are disabled when in noneditable OL list', () => {
+    const editor = hook.editor();
+    const content = '<ol contenteditable="false">\n' +
+      '<li contenteditable="true">editable</li>\n' +
+      '<li>noneditable</li>\n' +
+      '<li contenteditable="true">editable</li>\n' +
+    '</ol>';
+    editor.setContent(content);
+    TinySelections.setCursor(editor, [ 1, 2, 0 ], 0);
+    TinyUiActions.clickOnToolbar(editor, 'button[aria-label="Numbered list"]');
+    TinyAssertions.assertCursor(editor, [ 0, 2, 0 ], 0);
+    TinyAssertions.assertContent(editor, content);
+    TinyUiActions.clickOnToolbar(editor, 'button[aria-label="Bullet list"]');
+    TinyAssertions.assertCursor(editor, [ 0, 2, 0 ], 0);
+    TinyAssertions.assertContent(editor, content);
+  });
+
+  it('TINY-8920: List toolbar buttons are disabled when in noneditable UL list', () => {
+    const editor = hook.editor();
+    const content = '<ul contenteditable="false">\n' +
+      '<li contenteditable="true">editable</li>\n' +
+      '<li>noneditable</li>\n' +
+      '<li contenteditable="true">editable</li>\n' +
+    '</ul>';
+    editor.setContent(content);
+    TinySelections.setCursor(editor, [ 1, 2, 0 ], 0);
+    TinyUiActions.clickOnToolbar(editor, 'button[aria-label="Numbered list"]');
+    TinyAssertions.assertCursor(editor, [ 0, 2, 0 ], 0);
+    TinyAssertions.assertContent(editor, content);
+    TinyUiActions.clickOnToolbar(editor, 'button[aria-label="Bullet list"]');
+    TinyAssertions.assertCursor(editor, [ 0, 2, 0 ], 0);
+    TinyAssertions.assertContent(editor, content);
+  });
+
+  it('TINY-8920: List commands are disabled in noneditable OL list', () => {
+    const editor = hook.editor();
+    const content = '<ul contenteditable="false">\n' +
+      '<li contenteditable="true">editable</li>\n' +
+      '<li>noneditable</li>\n' +
+      '<li contenteditable="true">editable</li>\n' +
+    '</ul>';
+    editor.setContent(content);
+    TinySelections.setCursor(editor, [ 1, 2, 0 ], 0);
+    editor.execCommand('RemoveList');
+    TinyAssertions.assertCursor(editor, [ 0, 2, 0 ], 0);
+    TinyAssertions.assertContent(editor, content);
+    editor.execCommand('InsertUnorderedList');
+    TinyAssertions.assertCursor(editor, [ 0, 2, 0 ], 0);
+    TinyAssertions.assertContent(editor, content);
+    editor.execCommand('InsertOrderedList');
+    TinyAssertions.assertCursor(editor, [ 0, 2, 0 ], 0);
+    TinyAssertions.assertContent(editor, content);
+    editor.execCommand('InsertDefinitionList');
+    TinyAssertions.assertCursor(editor, [ 0, 2, 0 ], 0);
+    TinyAssertions.assertContent(editor, content);
+    editor.execCommand('mceListProps');
+    TinyAssertions.assertCursor(editor, [ 0, 2, 0 ], 0);
+    TinyAssertions.assertContent(editor, content);
```

Comment:
I think we should also test command `mceListUpdate`

Reply:
I'm not sure what the expected behaviour of this would be, since this command is able to change attributes, including `contenteditable`. I am leaning towards it still working on cef content, so I can add a test which confirms that this command still works?

Reply:
The list dialog definitely shouldn't work if the list itself is noneditable. As that would be the same as changing the list type via the advlist buttons for example.

Reply:
Updated to prevent `mceListUpdate` on non-editable lists.